test/encryption/encryption.test.js

```diff
@@ -22,43 +33,40 @@ function isEncryptedValue(object, property) {
 }
 
 describe('encryption integration tests', () => {
-  const cachedUri = process.env.MONGOOSE_TEST_URI;
-  const cachedLib = process.env.CRYPT_SHARED_LIB_PATH;
-
-  before(function() {
-    const cwd = process.cwd();
-    const file = fs.readFileSync(cwd + '/data/mo-expansion.yml', { encoding: 'utf-8' }).trim().split('\n');
-
-    // matches `key="value"` and extracts key and value.
-    const regex = /^(?<key>.*): "(?<value>.*)"$/;
-    const variables = Object.fromEntries(file.map((line) => regex.exec(line.trim()).groups).map(({ key, value }) => [key, value]));
+  /** @type { string } */
+  let cryptSharedLibPath;
+  /** @type { string } */
+  let clusterUri;
+  before(async function() {
+    const expansionFile = join(__dirname, '../..', 'fle-cluster-config.json');
```

Comment:
Because we now use `npm` packages to launch clusters, it would be straightforward to setup and teardown the clusters as a part of the test suite.  I considered adding this and decided against it, just because the existing usage pattern works well and means you don't need to setup/teardown a cluster per test run locally, but I can add this functionality if it is desirable.